data_prep/reuters_text.py

```diff
@@ -118,6 +119,16 @@ def _prepare_split(docs, classes):
             labels.append(classes.index(clz))
 
         return texts, labels
+    
+    def get_collate_fn(self):
```

Comment:
I'm not sure if I understand the need for this function if we just return None anyways

Reply:
It is handled in ```train.py```. You're right, we could also return ```default_data_collator```.